app/src/main/res/layout/feeds_apply_action_fragment.xml

```diff
@@ -0,0 +1,59 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this file different to the other "apply action" fragment? Looks like it would be easier to just re-use the same layout.